app/src/main/java/com/android/sample/model/recipe/RecipesViewModel.kt

```diff
@@ -28,8 +30,21 @@ class RecipesViewModel(private val repository: RecipeRepository) : ViewModel() {
   val currentRecipe: StateFlow<Recipe?>
     get() = _currentRecipe
 
+  private val _nextRecipe = MutableStateFlow<Recipe?>(null)
+  val nextRecipe: StateFlow<Recipe?>
+    get() = _nextRecipe
+
   init {
-    // Removed the fetchRandomRecipes() call from the init block for testing purposes
+    viewModelScope.launch {
+      fetchRandomRecipes(3)
```

Comment:
Thank you for the review! The value 3 is used temporarily as a placeholder while we determine the optimal number of recipes to fetch at the same time. It allows us to experiment with the current logic before finalizing the fetch threshold.

I agree that we should avoid hardcoding this value. I'll move it to a constant (e.g., MINIMUM_RECIPES_BEFORE_FETCH) to make it clearer and easier to update later when we adjust the number.

Reply:
I took your comment into account.

Reply:
It was because to avoid the repository sending to much API call at the same time. The number of fetch has been improved to 5 in the meantime. But for the moment for test purposes i let this like this as doesn't affect the user for the M1.